src/core/lib/security/credentials/google_default/google_default_credentials.cc

```diff
@@ -314,18 +289,56 @@ grpc_channel_credentials* grpc_google_default_credentials_create() {
     g_metadata_server_available = is_metadata_server_reachable();
   }
   gpr_mu_unlock(&g_state_mu);
+}
+
+static void default_call_creds(
+    grpc_core::RefCountedPtr<grpc_call_credentials>* call_creds,
```

Comment:
Instead of returning this via an output parameter, just make it the return value.